hack/util.sh

```diff
@@ -25,6 +25,8 @@ KARMADA_GO_PACKAGE="github.com/karmada-io/karmada"
 
 MIN_Go_VERSION=go1.20.0
 
+DEFAULT_CLUSTER_VERSION="kindest/node:v1.27.3"
```

Comment:
It makes sense to take this as the single source of truth, so when next time we bump kind, we only need to make an update here. Is it the idea?

Reply:
Yes, that's right.